src/main/java/com/example/TechItEasy/controllers/TelevisionDatabase.java

```diff
@@ -0,0 +1,56 @@
+package com.example.TechItEasy.controllers;
+
+import com.example.TechItEasy.exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping(value = ("/database"))
+public class TelevisionDatabase {
+
+    private static final List<String> televisionDatabase = new ArrayList<>();
+
+    @GetMapping("/televisions")
+    public ResponseEntity<Object> getTelevisionDatabase() {
```

Comment:
Hier is de returntype niet `ResponseEntity<Television>` wat het wel moet zijn, laat ik aan jou. In het geval dat je er echt niet uit komt, (of verschillende situaties kunt hebben), dan kun je <Object> of <?> (de wildcard) gebruiken